lib/ransack.rb

```diff
@@ -2,11 +2,8 @@
 
 require 'ransack/configuration'
 
-if defined?(::Mongoid)
-  require 'ransack/adapters/mongoid/ransack/constants'
-else
-  require 'ransack/adapters/active_record/ransack/constants'
-end
+require 'ransack/adapters'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.